decidim-core/lib/decidim/loggable.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # A concern that adds logging capability to the given model. Including this
+  # allows `Decidim::ActionLog` instances related to this class to properly
+  # render.
+  #
+  # We encourage you to overwrite the `log_presenter_class_for` class method
+  # to return your custom presenter for the given log type.
+  #
+  # Example:
+  #
+  #     class MyModel < ApplicationRecord
+  #       include Decidim::Loggable
+  #     end
+  module Loggable
+    extend ActiveSupport::Concern
+
+    class_methods do
+      # Public: Finds the presenter class for the given log type.
+      #
+      # log - a symbol representing the log type.
+      #
+      # Returns a Class.
+      def log_presenter_class_for(_log)
+        Decidim::Log::BasePresenter
```

Comment:
What do you think about trying to guess the presenter class from the class name? If the naming follows a convention you could avoid having to define this method in each class.

Reply:
That's what I'm trying to avoid :(